index.html

```diff
@@ -1,18 +1,164 @@
     <title>Document</title>
-</head>
-<body>
+    <style>
+   
+      </style>
+  </head>
+  <body>
+    <div class="container">
+      <div class="display">
+        <input type="text" placeholder="0" class="screen" />
+      </div>
+      <div class="buttons">
 
+        <div class="row">
+          <a class="diff">DEG</a>
+          <a class="diff">F-E</a>
+        </div>
+        <div class="row">
+          <a class="diff">MC</a>
+          <a class="diff">MR</a>
+          <a class="diff">M+</a>
+          <a class="diff">M-</a>
+          <a class="diff">MS</a>
+        </div>
+        <div class="row">
+          <a class="diff">Trigonometry</a>
+          <a class="diff">Function</a>
+        </div>
+
+
+        <div class="row">
+          <button class="btn">2^nd</button>
+          <button onclick="screen.value +='3.14159265359'">π</button>
+          <button onclick="screen.value +='2.71828182846'">e</button>
+          <button onclick="screen.value=''">C</button>
+          <button onclick="backspace()" style="font-weight: bold;">
+            <
+          </button>
+        </div>
+
+        <div class="row">
+          <button onclick="screen.value = Math.pow(screen.value,2)">x^2</button>
+          <button onclick="oneDivide()">1/x</button>
+          <button onclick="positive()">|x|</button>
+          <button onclick="exp()">exp</button>
+          <button> mod</button>
+        </div>
+        <div class="row">
+          <button>2modx</button>
+          <button class="btn">(</button>
+          <button class="btn">)</button>
+          <button onclick="fact()">n!</button>
+          <button class="btn">÷</button>
+        </div>
+
+        <div class="row">
+          <button onclick="">x^y</button>
+          <button class="btn">7</button>
+          <button class="btn">8</button>
+          <button class="btn">9</button>
+          <button class="btn">x</button>
+        </div>
+
+        <div class="row">
+          <button>10^x</button>
+          <button class="btn">4</button>
+          <button class="btn">5</button>
+          <button class="btn">6</button>
+          <button class="btn">-</button>
+        </div>
+
+        <div class="row">
+          <button onclick="log()">log</button>
+          <button class="btn">1</button>
+          <button class="btn">2</button>
+          <button class="btn">3</button>
+          <button class="btn">+</button>
+        </div>
+        <div class="row">
+          <button>In</button>
+          <button class="btn">+/-</button>
+          <button class="btn">0</button>
+          <button class="btn">.</button>
+          <button
+            onclick="screen.value='=' + eval(screen.value )" 
+          style="background-color: rgba(224, 224, 224, 0.199); font-weight: bold  ;">
+          =
+          </button>
+        </div>
+      </div>
+    </div>
 
 <script>
 
-    
-</script>
+var screen = document.querySelector('.screen');
```

Comment:
Move this JS code in a separate file named app.js or script.js and load that file here.

Reply:
okay